tests/test_sqlalchemy_controller_query_choices.py

```diff
@@ -0,0 +1,215 @@
+import pytest
+import sqlalchemy
+import flask
+import flask_restplus
+from layaberr import ValidationFailed
+
+import layabase
+
+
+@pytest.fixture
+def controller():
+    class TestTable:
+        __tablename__ = "test"
+
+        key = sqlalchemy.Column(sqlalchemy.String, primary_key=True)
+        test_field = sqlalchemy.Column(
+            sqlalchemy.String,
+            info={"marshmallow": {"choices": ["chose1", "chose2"]}},
+        )
+
+    controller = layabase.CRUDController(TestTable)
+    layabase.load("sqlite:///:memory:", [controller])
+    return controller
+
+
+@pytest.fixture
+def app(controller):
+    application = flask.Flask(__name__)
+    application.testing = True
+    api = flask_restplus.Api(application)
+    namespace = api.namespace("Test", path="/")
+
+    controller.namespace(namespace)
+
+    @namespace.route("/test")
+    class TestResource(flask_restplus.Resource):
```

Comment:
I don't see anything related to this pr in post/put or something else than parsers. Can you explain what is the purpose of all those endpoints ? Also I don't understand the post/put parser tests